src/items/itemDialog.component.tsx

```diff
@@ -285,13 +298,23 @@ function ItemDialog(props: ItemDialogProps) {
           shouldValidate: true,
         });
       }
+    } else if (isPrivilegedUser) {
+      ItemDetailsStepFormMethods.setValue(
+        'usage_status_id',
+        selectedItem?.usage_status_id ?? '', // sets to current usage status if editing
```

Comment:
```suggestion
        selectedItem?.usage_status_id ?? '', // sets to current usage status if editing
```

is this just for edit ? , there is no extra check preventing this from happening for duplicate . Can this comment be change please as it work as i expected for both duplicate and edit